docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -127,13 +188,36 @@ Get the file ``bigchaindb-dep.yaml`` from GitHub using:
 
    $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/bigchaindb/bigchaindb-dep.yaml
 
-Note that we set the ``BIGCHAINDB_DATABASE_HOST`` to ``mdb`` which is the name
-of the MongoDB service defined earlier.
 
-Edit the ``BIGCHAINDB_KEYPAIR_PUBLIC`` with the public key of this instance,
-the ``BIGCHAINDB_KEYPAIR_PRIVATE`` with the private key of this instance and
-the ``BIGCHAINDB_KEYRING`` with a ``:`` delimited list of all the public keys
-in the BigchainDB cluster.
+* Set ``metadata.name`` and ``spec.template.metadata.labels.app`` to the
+  value set in ``bdb-instance-name`` in the ConfigMap, followed by
+  ``-dep``.
+  For example, if the value set in the
+  ``bdb-instance-name`` is ``bdb-instance-0``, set the fields to the
+  value ``bdb-insance-0-dep``.
+
+* Set the value of ``BIGCHAINDB_KEYPAIR_PRIVATE`` (not base64-encoded).
+  (In the future, we'd like to pull the BigchainDB private key from
+  the Secret named ``bdb-private-key``, but a Secret can only be mounted as a file,
+  so BigchainDB Server would have to be modified to look for it
+  in a file.)
+
+* As we gain more experience running BigchainDB in testing and production,
+  we will tweak the ``resources.limits`` values for CPU and memory, and as
+  richer monitoring and probing becomes available in BigchainDB, we will
+  tweak the ``livenessProbe`` and ``readinessProbe`` parameters.
+
+* Set the ports to be exposed from the pod in the
+  ``spec.containers[0].ports`` section. We currently expose 2 ports -
+  ``bigchaindb-api-port`` and ``bigchaindb-ws-port``. Set them to the
+  values specified in the ConfigMap.
+
```

Comment:
Is it possible to link the entire section (from line 193 to 213) to the main document and just add a comment about the keyring changes here? I see that all the steps are the same from the first doc.